apps/v2/src/modules/blog/posts/posts.schema.ts

```diff
@@ -5,7 +5,7 @@ import { mediaPropertiesWithErrors, profileCore } from "../../auth/auth.schema"
 
 const POST_TITLE_MIN_LENGTH = 1
 const POST_TITLE_MAX_LENGTH = 280
-const POST_BODY_MAX_LENGTH = 2000
+const POST_BODY_MAX_LENGTH = 10000
```

Comment:
```suggestion
const POST_BODY_MAX_LENGTH = 10_000
```

Reply:
How could you do this

Reply:
I swear, I wrote it like that initially and I went back and removed it 😆